server/index.js

```diff
@@ -0,0 +1,12 @@
+const app = require('./app/index');
+const { db } = require('./db/index');
+
+const PORT = process.env.PORT || 3000;
+
+const force = false;
```

Comment:
consider using process variable instead of defining here (process.env.FORCE etc.)